cluster/images/etcd/rollback/etcd237/pkg/fileutil/fileutil.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
I agree that it shouldn't live in etcd repo.

@mml - can you please talk with people from MTV to check how this can be solved? Maybe @thockin has some ideas?